lib/matplotlib/axes/_base.py

```diff
@@ -4141,22 +4149,37 @@ def get_tightbbox(self, renderer, call_axes_locator=True,
             self.apply_aspect()
 
         if self.axison:
-            bb_xaxis = self.xaxis.get_tightbbox(renderer)
+            igl = 'x' if for_layout_only else None
+            try:
+                bb_xaxis = self.xaxis.get_tightbbox(renderer, ignore_label=igl)
+            except TypeError as e:
```

Comment:
`e` is unused.

Reply:
I don’t know much about the error handling but thought we needed this to get the stack trace correct.  But if you think it’s not needed, happy to remove it

Reply:
Only if you do `raise ... from e`.